lib/travis/api/app/endpoint/authorization.rb

```diff
@@ -339,6 +339,7 @@ def fetch
           def ensure_token_is_available
             unless user.tokens.first
               user.create_a_token
+              user.create_initial_subscription
```

Comment:
@piotrmilcarz, is this code used for new VCS login flow?

Reply:
Thank you for your question. I refactored it.